sycl/test/basic_tests/diagnostics/non-uniform-wk-gp-test.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// XFAIL: opencl
+// REQUIRES: level_zero
```

Comment:
why are other backends filtered?

Reply:
AFAIK, OpenCL 2.1 allows non uniform work-group sizes.

Reply:
This is a negative test to catch a specific error message happening with the level zero backend only.

Reply:
Where is the specific message being checked?

Reply:
The message is on line 35:
if (std::string(E.what()).find("Specified local size doesn't match the required work-group size specified in the program source") != std::string::npos)

Reply:
indeed, thanks